webextension/scripts/bulk-save.js

```diff
@@ -70,11 +70,16 @@ function importAllBookmarks() {
     chrome.bookmarks.getTree((nodeTree) => {
       nodeTree.forEach((node) => {
         processTreeNode(node)
+        if (count === 0) {
```

Comment:
Instead of storing the count variable, you could check the length by `Object.keys(bulkSaveObj).length`. We already have a `bulkSaveObj` where we are storing the URLs

Reply:
@anishsarangi Thank you for guiding me